src/app/app.routing.ts

```diff
@@ -7,6 +7,11 @@ import { LoginComponent } from './authentication/login.component';
 import { ForgotPasswordComponent } from './authentication/forgot-password.component';
 import { ResetPasswordComponent } from './authentication/reset-password.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
+import { SplashscreenComponent } from './splashscreen/splashscreen.component';
```

Comment:
It is a component added in PR #19. Those files are not included in this PR, that is why it shows it as an unused variable, but it is required for functionality.